angelsaddons-cab/control.lua

```diff
@@ -32,9 +33,8 @@ function setOnTickState(status)
       script.on_event(defines.events.on_tick, cab.tick)
       global.vehicleData.onTickActive = true
     end
-
   else -- status == false -- deactivate onTick
-    if (not global.vehicleData.deployedCabs) or (not next(global.vehicleData.deployedCabs)) then
+    if not global.vehicleData.deployedCabs or (not next(global.vehicleData.deployedCabs)) then
```

Comment:
This seems kinda dangerous...
In this specific case this is ok, but seems dangerous to get different behaviour...

Reply:
I get what you mean with dangerous. In my opinion brackets would help readability. 
It seems correct, and the formatter doesn't give any options to change this.
If we don't trust a formatter, we shouldn't use it. 🗡️ :D